app/services/subscriptions/dates/yearly_service.rb

```diff
@@ -78,9 +78,9 @@ def compute_previous_beginning_of_period(date)
       end
 
       def previous_anniversary_day(date)
-        year = date.month < subscription_date.month ? date.year - 1 : date.year
-        month = subscription_date.month
-        day = subscription_date.day
+        year = date.month < subscription_at.month ? date.year - 1 : date.year
```

Comment:
Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.